Src/Campus.Domain.Core/Models/AppUser.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace Campus.Domain.Core.Models
+{
+    public class AppUser
+    {
+        public int Id { get; set; }
+        
+        public string Name { get; set; }
+
+        public string Surname { get; set; }
+
+        public string Email { get; set; }
+
+        public string Login { get; set; }
+
+        public string PasswordHash { get; set; }
```

Comment:
Using password hash is not enough, you need to add password salt to strengthen auth security